app/views/container_service/show.html.haml

```diff
@@ -1,5 +1,7 @@
 - if %w(container_groups container_services container_routes).include?(@display)
   = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_nodes'.eql?(@display) && @showtype != "compare"
+  = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
Since this haml and `app/views/container_replicator/show.html.haml` have NO direct HTML output, this is simply some Ruby code to figure out what to render.  Suggest replacing these HAMLs with View Helpers as in https://github.com/ManageIQ/manageiq/pull/3645